contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -608,7 +608,7 @@ In time, it is our intent to add or autogenerate a sample viper configuration th
 
 With introducing Pod Security admission in Kubernetes by default, it is desired to execute e2e tests within bounded pod security policy levels. The default pod security policy in e2e tests is [restricted](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels). This is set in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go. This ensures that e2e tests follow best practices for hardening pods by default.
 
-Two helper functions are available for returning a minimal [restricted pod security context](https://github.com/kubernetes/kubernetes/blob/d7e6eab87d0fd005b238e3ec9b088e37d41a15d3/test/e2e/framework/pod/utils.go#L119) and a [restricted container security context](https://github.com/kubernetes/kubernetes/blob/d7e6eab87d0fd005b238e3ec9b088e37d41a15d3/test/e2e/framework/pod/utils.go#L127). These can be used to initialize pod or container specs to ensure adherence for the most restricted pod security policy.
+Two helper functions are available for returning a minimal [restricted pod security context](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/utils.go#L101) and a [restricted container security context](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/utils.go#L114). These can be used to initialize pod or container specs to ensure adherence for the most restricted pod security policy.
```

Comment:
Please use permalinks (associated with a git SHA) rather than linking to the master branch. These links will break or have incorrect line numbers if the files are moved or changed.